project_1.py

```diff
@@ -0,0 +1,59 @@
+# opening and getting variables
+print("Hello, This is my final project")
+user_name = input("What is your name? ")
+print("Hi " + user_name +", nice to meet you")
+print("This is a special calculator, I would need two numbers from you")
+number_1 = int(input("Enter first number: "))
+number_2 = int(input("Enter second number: "))
+print("Thank you for putting in your numbers, " + str(number_1) + " and " + str(number_2))
+
+# checking and saying to the user every number if he even or odd
+if number_1 % 2 == 0:
+    print("I can see that the first number is even")
```

Comment:
שים לב שהמשפט "I can see that the first number is  " מופיע גם בשורה 12 וגם בשורה 14. זה שכפול קוד, זה לא טוב. צריך לתקן את זה. תראה בבקשה את השיעור על שכפולי קוד כדי לראות איך מסדרים את זה